mvlearn/embed/kcca.py

```diff
@@ -3,20 +3,45 @@
 ====================================
 Python module for kernel canonical correlation analysis (kCCA)
 
-Code modified from UC Berkeley, Gallant lab
-(https://github.com/gallantlab/pyrcca)
-Copyright 2016, UC Berkeley, Gallant lab.
+Adopted from Steven Van Vaerenbergh's MATLAB Package 'KMBOX'
+https://github.com/steven2358/kmbox
 """
 
 from .base import BaseEmbed
 from ..utils.utils import check_Xs
 
 import numpy as np
-from scipy.linalg import eigh
+import numpy.matlib
+from scipy import linalg
 
 
 class KCCA(BaseEmbed):
-    """
+    r"""
+    The kernel canonical correlation analysis (KCCA) is a method
+    that generalizes the classical linear canonical correlation
+    analysis (CCA) to nonlinear setting.  It allows us to depict the
+    nonlinear relation of two sets of variables and enables
+    applications of classical multivariate data analysis
+    originally constrained to linearity relation (CCA).
+
```

Comment:
Can you include the actual citation here? Follow the example in mv_kmeans (https://github.com/neurodata/mvlearn/blob/kcca/mvlearn/cluster/mv_kmeans.py) from line 28 and line 129 so that sphinx actually links the two. This way, the references in the documentation get ordered correctly. (Call your references like [#1KCCA] and [#2KCCA] for example, and in the Notes section somewhere, cite them with [#1KCCA]_ and [#2KCCA]_

Reply:
will work on this